packages/mobile/src/account/FiatExchange.tsx

```diff
@@ -24,6 +24,10 @@ function FiatExchange() {
     navigation.navigate(Screens.FiatExchangeAmount, { isAddFunds: true })
   }
 
+  function goToExternalExchanges() {
+    navigation.navigate(Screens.ExternalExchanges, { isAddFunds: false })
```

Comment:
I think the `isAddFunds` doesn't belong here

Reply:
Thanks for catching this!